test/luatest_helpers/server.lua

```diff
@@ -27,12 +27,15 @@ local DEFAULT_CHECKPOINT_PATTERNS = {"*.snap", "*.xlog", "*.vylog",
 -- * 'alias' is mandatory.
 -- * 'command' is optional, assumed test/instances/default.lua by
 --   default.
+-- * 'copydir' is optional, specifies a directory: if specified, the directory's
+--   contents will be recursively copied into 'workdir' during initialization.
```

Comment:
Nit: confusing name. Maybe `datadir` or `srcdir`? Not sure. I won't mind if you leave it as is though.

Reply:
Fixed: changed name to `datadir`.